xarray/core/common.py

```diff
@@ -1188,6 +1188,7 @@ def where(self, cond, other=dtypes.NA, drop: bool = False):
 
         Examples
         --------
+        >>> import xarray as xr
```

Comment:
I had thought we didn't do this by convention (neither for numpy, which is in the line below, contradicting that convention), because it would be everywhere.

Is that the case? I don't have a particularly strong view either way, though we could decide on the convention now.

Reply:
that's what I was thinking, too: technically we don't need any imports of `numpy`, `pandas` or `xarray`: https://github.com/pydata/xarray/blob/351b0aac671a910803dcf618a18bb33cc8f8f2ab/conftest.py#L26-L41 so I would lean towards removing all imports to those. That said, it would probably be good to have some obvious part of the docs (`api.rst`?) explain that all examples assume these imports.